File: image-train-filters-app/src/main/java/org/jesperancinha/itf/android/config/ControlConfiguration.java
package org.jesperancinha.itf.android.config;

import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import org.jesperancinha.itf.android.common.ChartizateSurfaceView;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@AllArgsConstructor
@Builder
public class ControlConfiguration {
    private final ChartizateSurfaceView svSelectedColor;

    private final EditText editFontSize;

    private final Button btnStart;

    private final Button btnStartEmail;

    private final TextView textStatus;
}

File: app-g2/app/src/main/java/org/jesperancinha/itf/app/config/ControlConfiguration.kt
package org.jesperancinha.itf.app.config

import android.widget.Button
import android.widget.Spinner
import android.widget.TextView
import org.jesperancinha.itf.app.common.ChartizateSurfaceView


class ControlConfiguration (
    val svSelectedColor: ChartizateSurfaceView,
    val btnStart: Button,
    val btnStartEmail: Button,
    val textStatus: TextView,
    val spiFontType: Spinner,
    val spiDistribution: Spinner,
    val spiLanguageCode: Spinner,
)